### PR TITLE
Deleted GatingGrowl from Sublayout

### PR DESCRIPTION
**Pull request by lzach83**
## Link to Issue
Closes: #6516 

## Description of Changes
- Delete `GatingGrowl` component from `Sublayout` to ensure it no longer shows

## "How We Fixed It"

- Removed the import and usage of `<GatingGrowl />`

## Test Plan
- Ensure your localStorage `dontShowGatingGrowlEver` is set to false or deleted
- Navigate to a community
- Ensure the gating growl does not show

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/Sublayout.tsx [hunk 1]
```diff
@@ -13,7 +13,6 @@ import { SublayoutBanners } from './SublayoutBanners';
 import { SublayoutHeader } from './SublayoutHeader';
 import { AdminOnboardingSlider } from './components/AdminOnboardingSlider';
 import { Breadcrumbs } from './components/Breadcrumbs';
-import GatingGrowl from './components/GatingGrowl/GatingGrowl';
```
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/Sublayout.tsx:16]: lets delete this component and all its zustand state code as well

Comment (lzach83) [reply packages/commonwealth/client/scripts/views/Sublayout.tsx:16]: All set

Comment (CowMuon) [reply packages/commonwealth/client/scripts/views/Sublayout.tsx:16]: This is a great catch @mzparacha. I almost wrote in the ticket to make sure we're cleaning up any no longer needed code (incl zustand)

File: packages/commonwealth/client/scripts/views/components/GatingGrowl/GatingGrowl.tsx [hunk 2]
```diff
@@ -1,101 +0,0 @@
...
-  return (
-    <CWGrowl disabled={disabled} position="bottom-right">
-      <div className="GatingGrowl">
-        <CWIconButton
-          iconName="close"
-          iconSize="medium"
-          className="closeButton"
-          onClick={handleExit}
-        />
-        <img src="../../../static/img/groupGrowl.png" alt="" className="img" />
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/GatingGrowl/GatingGrowl.tsx:44]: remove groupGrowl.png image as well

**Review by masvelio — APPROVED** [review]

**Review by jnaviask — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/components/GatingGrowl/GatingGrowl.tsx (hunk 2 above)
Comment (lzach83) [reply packages/commonwealth/client/scripts/views/components/GatingGrowl/GatingGrowl.tsx:44]: All set on this

**Review by CowMuon — APPROVED** [review]
GTG → v0.9.5

**Pull request merged**